<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modPwDialog</class>
 <widget class="QDialog" name="modPwDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生 - 修改密码</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>登录名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>旧密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>新密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>再输一遍：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="oldPwLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPwLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="rpPwLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="oldPwErrTip">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>*密码输入错误，请重试！</string>
   </property>
  </widget>
  <widget class="QLabel" name="rpPwErrTip">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>*两次新密码输入不一致
请重试！</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OkBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>oldPwLineEdit</tabstop>
  <tabstop>newPwLineEdit</tabstop>
  <tabstop>rpPwLineEdit</tabstop>
  <tabstop>OkBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
